src/extensions/ckb_balance/mod.rs

```diff
@@ -17,20 +17,32 @@ use rlp::{Decodable, Encodable, Rlp};
 use std::collections::HashMap;
 use std::sync::Arc;
 
+pub const SECP256K1_BLAKE160: &str = "secp256k1_blake160";
+
 pub struct CkbBalanceExtension<S, BS> {
     store: S,
     indexer: Arc<Indexer<BS>>,
     _net_ty: NetworkType,
-    _config: HashMap<String, DeployedScriptConfig>,
+    config: HashMap<String, DeployedScriptConfig>,
 }
 
 impl<S: Store, BS: Store> Extension for CkbBalanceExtension<S, BS> {
     fn append(&self, block: &BlockView) -> Result<()> {
         let mut ckb_balance_map = CkbBalanceMap::default();
         let mut ckb_balance_change = ckb_balance_map.inner_mut();
+        let config = self.config.get(SECP256K1_BLAKE160).unwrap();
+
+        if block.is_genesis() {
+            let txs = block.transactions();
+            let genesis_cellbase_tx = txs.get(0).unwrap();
+            for output in genesis_cellbase_tx.outputs().into_iter() {
+                if is_secp256k1_blake160_cell(&output, &config.script) {
+                    self.change_ckb_balance(&output, &mut ckb_balance_change, false);
+                }
+            }
+        }
```

Comment:
There are two problems here.
1. All the cells in the genesis must be normal ckb cell, so the judgement is useless.
2. It will be re-indexed the cellbase again at line 70.

Reply:
1. not every cells in the gensis must be normal ckb cell(with lock_script as `secp256k1_blake160`), [ref](https://explorer.nervos.org/transaction/0xe2fb199810d49a4d8beec56718ba2593b665db9d52299a0f9e6e75416d73ff5c);
2. fixed;